### PR TITLE
199 cordex xfails

### PR DESCRIPTION
**Pull request by charles-turner-1**
- Add xfails on cordex translator (broken - see #199).
- Update storage flags, correct package dir
-  'Robustify' a couple of tests

### CODE REVIEW

File: bin/test_end_to_end.sh [hunk 1]
```diff
@@ -1,11 +1,11 @@
 #!/bin/bash -l
 
 #PBS -P iq82
-#PBS -l storage=gdata/xp65+gdata/ik11+gdata/cj50+gdata/p73+gdata/dk92+gdata/al33+gdata/rr3+gdata/fs38+gdata/oi10
+#PBS -l storage=gdata/xp65+gdata/ik11+gdata/cj50+gdata/p73+gdata/dk92+gdata/al33+gdata/rr3+gdata/fs38+gdata/oi10+gdata/hq89+gdata/py18+gdata/ig45+gdata/zz63+gdata/rt52
```
Comment (marc-white) [inline bin/test_end_to_end.sh:4]: I'm not sure why this is flagging as a change, given these updates already exist in `main`... is this branch up-to-date with `main`?

Comment (charles-turner-1) [reply bin/test_end_to_end.sh:4]: This is on the e2e, not the build_all script - I've just updated the flags on `test_end_to_end.sh` to match `build_all.sh`

Comment (marc-white) [reply bin/test_end_to_end.sh:4]: You're right, so it is. Ignore me.

**Review by marc-white — APPROVED** [review]
Looks good to go, although it looks like e2e hasn't passed on this branch yet.

**Comment by charles-turner-1** [comment]
> Looks good to go, although it looks like e2e hasn't passed on this branch yet.

Haven't updated the reference copy yet - good catch.

**Comment by charles-turner-1** [comment]
I'm still unable to update the reference repo at `/g/data/xp65/admin/access-nri-intake-catalog` & I've broken my git keys trying to fix it (see unverified badge above). I think that merging this & updating the reference repo should fix the issue.

NB. Last failed CI run actually passed: see

```bash
$ cd /g/data/xp65/admin/access-nri-intake-catalog 
$ cat test_end_to_end.sh.o130485707
...    array = array.get_duck_array()

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========== 108 passed, 1 xfailed, 57 warnings in 542.74s (0:09:02) ============

======================================================================================
                  Resource Usage on 2024-12-11 17:14:31:
   Job Id:             130485707.gadi-pbs
   Project:            iq82
   Exit Status:        0
   Service Units:      5.02
   NCPUs Requested:    12                     NCPUs Used: 12              
                                           CPU Time Used: 00:37:51        
   Memory Requested:   64.0GB                Memory Used: 64.0GB          
   Walltime requested: 00:30:00            Walltime Used: 00:09:25        
   JobFS requested:    100.0MB                JobFS used: 3.13MB          
======================================================================================
```

@rbeucher any ideas as to why this might be failing?

**Comment by charles-turner-1** [comment]
@marc-white @rbeucher 

 I think you may have somehow set some of the git refs to read-only for anyone but you - see
```bash
$ cd /g/data/xp65/admin/access-nri-intake-catalog/.git/logs/refs/remotes/origin 

[ct1163@gadi-login-01 origin]$ ll -t
total 180
-rw-rw-r--+ 1 ct1163 xp65_w  483 Dec 11 18:11 199-cordex-xfails
-rw-r--r--  1 mcw120 xp65_w  307 Dec 11 16:22 271-consolidation-on-pathlib-module *
-rw-r--r--  1 mcw120 xp65_w  308 Dec 11 16:22 pre-commit-ci-update-config *
-rw-rw-r--+ 1 ct1163 xp65_w  161 Dec 11 15:18 257-replace-mocks
-rw-rw-r--+ 1 ct1163 xp65_w  161 Dec 11 15:18 279-documentation-updates
-rw-rw-r--+ 1 ct1163 xp65_w  161 Dec 11 15:18 285-add-metadata-validation-tests
-rw-rw-r--+ 1 ct1163 xp65_w  161 Dec 11 15:18 286-bug-metadata-issues-in-v100b2
-rw-rw-r--+ 1 ct1163 xp65_w  161 Dec 11 15:18 290-align-test-datasets
-rw-rw-r--+ 1 ct1163 xp65_w  161 Dec 11 15:18 292-flaky-tests
-rw-rw-r--+ 1 ct1163 xp65_w  161 Dec 11 15:18 294-enable-catalog-version-reporting
-rw-rw-r--+ 1 ct1163 xp65_w  161 Dec 11 15:18 297-translator-errors
-rw-rw-r--+ 1 ct1163 xp65_w  161 Dec 11 15:18 299-doc-updates
-rw-rw-r--+ 1 ct1163 xp65_w  161 Dec 11 15:18 303-sanity-check-ipynb
-rw-rw-r--+ 1 rb5533 xp65_w  980 Dec 11 15:18 main
-rw-rw-r--+ 1 rb5533 xp65_w  322 Dec  3 11:16 153-e2e
-rw-rw-r--+ 1 rb5533 xp65_w  322 Dec  3 11:16 168-access-cm3-builder
-rw-rw-r--+ 1 rb5533 xp65_w  483 Dec  3 11:16 175-data-request-add-om4_025jra_ryf
-rw-r--r--  1 mcw120 xp65_w  154 Dec  3 11:16 265-bug-cli-tests-fail-if-not-top-level-directory *
-rw-r--r--  1 mcw120 xp65_w  154 Dec  3 11:16 270-teardown-of-temporary-file-structures *
-rw-r--r--  1 mcw120 xp65_w  154 Dec  3 11:16 273-remove-hh5-references *
-rw-r--r--  1 mcw120 xp65_w  154 Dec  3 11:16 277-v100-release *
-rw-r--r--  1 mcw120 xp65_w  154 Dec  3 11:16 283-bug-cordex-build-failing-in-v100b2 *
drwxr-sr-x  3 mcw120 xp65_w 4096 Dec  3 11:16 dependabot
-rw-r--r--  1 mcw120 xp65_w  154 Dec  3 11:16 rbeucher-patch-4 *
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 184-data-request-add-cosima-experiments
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 188-the-frequency-in-the-intake-catalog-does-not-account-for-snapshot-data
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 191-default-to-a-latest-catalog-version
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 216-make-metadata-validate-provide-a-full-report-on-first-pass
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 223-model-required
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 224-remove-missing-erai-metadatayaml
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 235-config
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 246-existing-catalog-that-doesnt-conform
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 254-deprecate-py39
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 263-bug-build-function-minmax
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 issue176-om3-regex
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 rbeucher-patch-1
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 rbeucher-patch-2
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 rbeucher-patch-3
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 toggle-userdata
-rw-rw-r--+ 1 rb5533 xp65_w  168 Nov 21 12:40 164-update-metadata-schema-exp-1-0-3
-rw-rw-r--+ 1 rb5533 xp65_w  168 Oct  8 16:41 196-data-request-cmip6-era-regional-climate-model-data-ccam-acs
-rw-rw-r--+ 1 rb5533 xp65_w  330 Oct  8 16:41 660-coordinate-variables
-rw-rw-r--+ 1 rb5533 xp65_w  168 Oct  8 16:41 barpa_py18
-rw-rw-r--+ 1 rb5533 xp65_w  161 Oct  3 11:26 fix_metadata_yml
-rw-rw-r--+ 1 rb5533 xp65_w  206 Oct  1 12:01 HEAD
```

I've starred lines where permissions have gone awry... not quiet sure how this has happened but it seems to be whats blocking this.

**Comment by rbeucher** [comment]
It's @marc-white who is blocking, right? 😂 Yeah collaborative work on a local git repo is not a great idea...

**Comment by charles-turner-1** [comment]
Yup, tricky business, haha... Is it safe to delete & reclone the repo or is that going to make the situation worse rather than better?

**Comment by rbeucher** [comment]
I'll have to fix that tomorrow. Laptop is in the office.

**Comment by marc-white** [comment]
I think you should be fine to just destroy the directory and re-clone it.

**Comment by charles-turner-1** [comment]
I'm satisfied that these tests run fine when called directly on Gadi - shall we merge this & open an issue for fixing the workflow?

**Pull request merged**